WindowsServerDocs/storage/storage-spaces/troubleshooting-storage-spaces.md

```diff
@@ -415,6 +415,7 @@ In a Windows Server 2016 Storage Spaces Direct cluster, you might see the Health
 "Removing from Pool" is an intent set when **Remove-PhysicalDisk** is called but stored in Health to maintain state and allow recovery if the remove operation fails. You can manually change the OperationalStatus to Healthy with one of the following methods:
 
 - Remove the physical disk from the pool, and then add it back.
+- Import-Module Clear-PhysicalDeskHealthData.ps1
```

Comment:
❗ Typo warning: -Desk- should be **Disk** in this file name.
```suggestion
- Import-Module Clear-PhysicalDiskHealthData.ps1
```
See also the related issue ticket #3309 (**Clear-PhysicalDiskHealthData.ps1 - make sure you import-module first**) for details.